source/_components/light.hue.markdown

```diff
@@ -29,12 +29,22 @@ light:
   host: DEVICE_IP_ADDRESS
 ```
 
+The light component supports multiple entries in <code>configuration.yaml</code> by appending a sequential number to the section: <code>light 2:</code>, <code>light 3:</code> etc.  When adding in a second hub, add the following lines to your `configuration.yaml`:
```

Comment:
Specifying multiple platforms for a component is Home Assistant 101, we should not mention this in the platform docs here.